src/utils/money.cljs

```diff
@@ -237,3 +238,7 @@
 
       :else
       (str amount))))
+
+(schema/=> format-amount
+  [:=> [:cat [:maybe :int]]
+   [:maybe :string]])
```

Comment:
FYI: The macro approach with `schema/=>` is more convenient when the function to instrument is not wrapped in higher-order functions (which is the case for quo components wrapped in `quo.theme/with-theme`).